lib/GADS/Layout.pm

```diff
@@ -311,11 +311,30 @@ sub _build_security_marking {
 
 sub _build_reports
 {   my $self = shift;
-    my $reports_rs = $self->schema->resultset('Report')->search({
-        instance_id => $self->instance_id,
-        deleted => undef
-    });
```

Comment:
Keep the above, and then chain on the searches as required (or not in the case of superadmin)